aptos-move/aptos-aggregator/src/types.rs

```diff
@@ -3,20 +3,24 @@
 
 use crate::bounded_math::SignedU128;
 use aptos_logger::error;
-use aptos_types::delayed_fields::{
-    bytes_and_width_to_derived_string_struct, derived_string_struct_to_bytes_and_length,
-    is_derived_string_struct_layout,
-};
 // TODO[agg_v2](cleanup): After aggregators_v2 branch land, consolidate these, instead of using alias here
-pub use aptos_types::delayed_fields::{
-    DelayedFieldID, PanicError, TryFromMoveValue, TryIntoMoveValue,
-};
+pub use aptos_types::delayed_fields::PanicError;
 use move_binary_format::errors::PartialVMError;
 use move_core_types::{
     value::{IdentifierMappingKind, MoveTypeLayout},
     vm_status::StatusCode,
 };
-use move_vm_types::values::{Struct, Value};
+pub use move_vm_types::delayed_values::delayed_field_id::DelayedFieldID;
```

Comment:
so we can now import aptos_aggregator::types::DelayedFieldID? nice.

Reply:
yes! But I have not made this uniform (yet).